deepmd/entrypoints/main.py

```diff
@@ -281,7 +281,7 @@ def main_parser() -> argparse.ArgumentParser:
         "--detail-file",
         type=str,
         default=None,
-        help="File where details of energy force and virial accuracy will be written",
+        help="Files where details of energy, force and virial accuracy/accuracy per atom will be written",
```

Comment:
Actually the argument assigns **the prefix of** files.

Reply:
So maybe the proper description is "The prefix to files where details of energy, force and virial accuracy/accuracy per atom will be written"?

Reply:
Agree

Reply:
ok I have changed the file